src/klein/_version.py

```diff
@@ -1,11 +1,11 @@
 """
-Provides klein version information.
+Provides Klein version information.
 """
 
 # This file is auto-generated! Do not edit!
-# Use `python -m incremental.update klein` to change this file.
+# Use `python -m incremental.update Klein` to change this file.
 
 from incremental import Version
 
-__version__ = Version("klein", 23, 5, 0)
+__version__ = Version("Klein", 23, 12, 0, release_candidate=7)
```

Comment:
do you need to remove the release candidate here?

Reply:
That's the final step in `release.py`.  It does an `python -m incremental.update --final` and that removes the `rc` from the version number, both here and in all the `@since`'s that are lying around throughout the codebase.

Reply:
Ah okay, in that case this looks good to me!

Reply:
want to make that official with an "approve"? :)